climakitae/selectors.py

```diff
@@ -721,10 +721,8 @@ def _display_select(selections, location, location_type = "area average"):
     for location_type -- 'area average' is the only choice, until station-based
     data are available.
     """
-    assert location_type in [
-        "area average",
-        "station",
-    ], "Please enter either 'area average' or 'station'."
+    if location_type not in ["area average", "station"]:
+        raise ValueError("Please enter either 'area average' or 'station'.")
```

Comment:
Glad you noted this as I've been meaning to come back to this. We don't actually ever use this code; I think its old placeholder code from many months ago. Should we just delete these lines?

Reply:
Yes! Reducing unneeded code would be great!

Reply:
Will do!